src/deepqmc/conf/ansatz/ferminet.yaml

```diff
@@ -40,7 +45,6 @@ omni_factory:
     edge_types:
```

Comment:
Could we maybe merge the `edge_types` with the `update_features`, now that we have a removed the GNN base class?

Reply:
Ok I see. Lets keep this for another PR.